packages/turbo-codemod/src/commands/migrate/steps/getTransformsForMigration.ts

```diff
@@ -13,13 +13,24 @@ function getTransformsForMigration({
   fromVersion: string;
   toVersion: string;
 }): Array<Transformer> {
-  const transforms = loadTransformers();
-  return transforms.filter((transformer) => {
+  const transforms = loadTransformers().filter((transformer) => {
     return (
       gt(transformer.introducedIn, fromVersion) &&
       lte(transformer.introducedIn, toVersion)
     );
   });
+
+  transforms.sort((a, b) => {
+    if (gt(a.introducedIn, b.introducedIn)) {
+      return 1;
+    } else if (eq(a.introducedIn, b.introducedIn)) {
+      return 0;
+    } else {
+      return -1;
+    }
+  });
```

Comment:
I can never remember what positive and negative means. Is this sorting so more recent transforms run first or vice versa? a comment would be useful if that's why we're sorting in the first place

Reply:
This sorts ascending; transforms run from oldest to newest, allowing us to rely on the previous transforms having already been completed.

As for remembering:
- JS sort is default ascending.
- 1 > 0, so return 1 when it is greater.
- 0 = 0, so map 0 to equal.
- -1 < 0, so -1 maps to smaller.

If you pass negative zero in, well, you get what was coming. (I don't know if JS cares or not.)

Reply:
😬 good catch!